Sources/Tracing/Docs.docc/Guides/ImplementATracer.md

```diff
@@ -0,0 +1,169 @@
+# Implement a Tracer
+
+## Overview
+
+This guide is aimed at ``Tracer`` and `Instrument` protocol implementation authors.
+
+This guide is for you if you find yourself in need of implementing your own tracing client such as Zipkin, Jaeger, X-Trace, OpenTelemetry or something similar that is custom to your company or distributed system. It will also complete your understanding of how distributed tracing systems actually work, so even the casual developer may find this guide useful to read through, even if not implementing your own tracers. 
+
+## Do you need an Instrument or a Tracer?
+
+Distributed tracing offers two types of instrumentation protocols: an instrument, and a tracer.
+
+A tracer is-an instrument as well, and further refines it with the ability to start a trace ``Span``.
+
+## Creating an instrument
+
+In order to implement an instrument you need to implement the `Instrument` protocol.
+`Instrument` is part of the `Instrumentation` library that `Tracing` depends on and offers the minimal core APIs that allow implementing instrumentation types.
+
+`Instrument` has two requirements:
+
+1. An `Instrument/extract(_:into:using:)` method, which extracts values from a generic carrier (e.g. HTTP headers) and store them into a `Baggage` instance
```

Comment:
I'm assuming this is referenced this way, so that when cross-module symbols are able to be referenced by DocC, we can just come back and add the 2nd set of `?

Reply:
Yeah currently we can't cross link so it's just a code marker. I may go around and make them explicit links manually hmm